src/schemas/specimen-collection.js

```diff
@@ -72,7 +72,8 @@ export const SPECIMEN_COLLECTION_SCHEMA = yup.object({
       "unknown",
       "other",
     ])
-    .default("unknown")
+    .default("null")
```

Comment:
We should use null instead of "null" since it is not a string.

Also in order to make it work with the schema validator it is recommended to mark the field as nullable so that it can be initialized to null otherwise it will throw an error. Learn more [here](https://github.com/jquense/yup#schemanullable-schema)

```js
.nullable()
.default(null)
```